src/core_modules/capture-core/components/Pages/MainPage/EventsList/Pagination/ListPagination.component.js

```diff
@@ -0,0 +1,33 @@
+// @flow
```

Comment:
_This comment applies for the whole file_

Simply renamed the `/capture-core/components/Pages/MainPage/EventsList/Pagination/ListPagination.js` and exporting it now as const instead of default. No real functional changes.